### PR TITLE
rename acm-controller dir to controllers

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: cmd/acm-controller/app/server.go [hunk 1]
```diff
@@ -7,15 +7,15 @@ import (
...
-	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/acm-controller/autodetect"
-	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/acm-controller/clusterinfo"
-	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/acm-controller/clusterrbac"
-	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/acm-controller/clusterrole"
-	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/acm-controller/gc"
-	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/acm-controller/inventory"
 	actionv1beta1 "github.com/open-cluster-management/multicloud-operators-foundation/pkg/apis/action/v1beta1"
 	clusterinfov1beta1 "github.com/open-cluster-management/multicloud-operators-foundation/pkg/apis/internal.open-cluster-management.io/v1beta1"
 	inventoryv1alpha1 "github.com/open-cluster-management/multicloud-operators-foundation/pkg/apis/inventory/v1alpha1"
+	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/controller/autodetect"
```
Comment (qiujian16) [inline cmd/acm-controller/app/server.go:13]: controllers?

Comment (zhiweiyin318) [reply cmd/acm-controller/app/server.go]: done

**Pull request merged**